src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/CheckedListBoxAccessibleObjectTests.cs

```diff
@@ -103,5 +103,66 @@ public void CheckedListBoxAccessibleObject_Role_IsExpected_ByDefault(bool create
             Assert.Equal(expectedRole, actual);
             Assert.Equal(createControl, checkedListBox.IsHandleCreated);
         }
+
+        [WinFormsTheory]
+        [InlineData(0)]
+        [InlineData(3)]
+        public void CheckedListBoxAccessibleObject_GetChildCount_ReturnsExpected(int childCount)
+        {
+            using CheckedListBox checkedListBox = new();
+
+            for (int i = 0; i < childCount; i++)
+            {
+                checkedListBox.Items.Add(i);
+            }
+
+            int actual = checkedListBox.AccessibilityObject.GetChildCount();
+
+            Assert.Equal(childCount, actual);
+            Assert.False(checkedListBox.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void CheckedListBoxAccessibleObject_RuntimeId_NotNull()
+        {
+            using CheckedListBox checkedListBox = new();
+
+            Assert.NotNull(checkedListBox.AccessibilityObject.RuntimeId);
+            Assert.False(checkedListBox.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void CheckedListBoxAccessibleObject_FragmentNavigate_NavigateToFirstChild_IsExpected()
+        {
+            using CheckedListBox checkedListBox = new();
+            AccessibleObject accessibleObject = checkedListBox.AccessibilityObject;
+
+            for (int i = 0; i < 3; i++)
+            {
+                checkedListBox.Items.Add(i);
+            }
```

Comment:
I think in this case we do not need a `for (int i = 0; i < 3; i++)`. Please fix it in the other tests
```suggestion
            checkedListBox.Items.Add(0);
            checkedListBox.Items.Add(1);
            checkedListBox.Items.Add(2);
```